File: Program17.c
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Problem Statement : Accept N numbers from user and return the largest number.
//	
//	Input		  :	N-6
//				Elements: 85	66	3	66	93	88
//
//	Output		  : 93
//	
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
#include<unistd.h>
#include<stdlib.h>

typedef unsigned int u_int;
typedef signed int s_int;

u_int findLargestElement(int iLength,int iArr[])
{

	auto s_int iLarge = 0,iCnt = 0;	
	
	for(iCnt = 0; iCnt < iLength; iCnt++)
	{
		iLarge = iArr[0];
		
		if(iArr[iCnt] > iLarge)
		{
			iLarge = iArr[iCnt];
			break;			
		}
		
	}
	
	return iLarge;

}

int main(void)
{
	u_int findLargestElement(int,int *);
	
	auto s_int iSize	 = 0;
	auto s_int iRet 	 = 0;
	register s_int iCnt	 = 0;	
	auto s_int *iPtr 	 = NULL;
	
	printf("Enter number of elements you want to store in array : \n");
	scanf("%d",&iSize);
	
	iPtr = (int *)malloc(iSize * sizeof(int));
	if(iPtr == NULL)
	{
		perror("malloc");
		_exit(EXIT_FAILURE);
	}		
	
	printf("\nEnter %d elements : \n",iSize);
	
	for(iCnt = 0; iCnt < iSize; iCnt++)
	{
		scanf("%d",&iPtr[iCnt]);
	}
	
	printf("\nElements stored in array are : \n");
	
	for(iCnt = 0; iCnt < iSize; iCnt++)
	{
		printf("%d\t",iPtr[iCnt]);
	}
	printf("\n");
	
	iRet = findLargestElement(iSize,iPtr);
	printf("\nLargest element in array is : %d\n",iRet);
	
	free(iPtr);
	
	exit(EXIT_SUCCESS);

}	

File: Program17.java
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Problem Statement : Accept N numbers from user and return the largest number.
//	
//	Input		  :	N-6
//				Elements: 85	66	3	66	93	88
//
//	Output		  : 93
//	
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class ArrayX
{
	private int iArr[];
	
	ArrayX(int iSize)
	{
		iArr  = new int[iSize];
	}
	
	public void acceptElements()
	{
		int iCnt = 0;
	
		Scanner sObj = new Scanner(System.in);
		
		System.out.println("\nEnter elements : ");
		
		for(iCnt = 0; iCnt < iArr.length; iCnt++)
		{
			iArr[iCnt] = sObj.nextInt();
		}
		
		sObj.close();
	}
	
	public void displayElements()
	{
		int iCnt = 0;
		
		System.out.println("\nElements stored in array are : ");
		
		for(iCnt = 0; iCnt < iArr.length; iCnt++)
		{
			System.out.print(iArr[iCnt]+"\t");
		}
		
		System.out.println();
	}
	
	public int findLargestElement()
	{
		int iCnt = 0,iLarge = 0;
		
		for(iCnt = 0; iCnt < iArr.length; iCnt++)
		{
			iLarge = iArr[0];
			
			if(iArr[iCnt] > iLarge)
			{
				iLarge = iArr[iCnt];
			}
		}	
		
		return iLarge;
	}

}

class Program17
{

	public static void main(java.lang.String Arg[])
	{
		int iLength = 0,iRet = 0;
		
		Scanner sObj = new Scanner(System.in);
		
		System.out.println("Enter number of elements that you want to store in array : ");
		iLength = sObj.nextInt();
		
		ArrayX aObj = new ArrayX(iLength);
		
		aObj.acceptElements();
		aObj.displayElements();
		
		iRet = aObj.findLargestElement();
		System.out.println("\nLargest element from array is : "+iRet);
		
		sObj.close();			
	}

}
